Jenkinsfile

```diff
@@ -19,7 +19,7 @@ app.build(
 
   stage('Code Checkout') {
     scmVars = checkout scm
-    appImage = "quay.io/powerhome/playbook:${git.triggeringCommit(scmVars)}"
+    appImage = "image-registry.powerapp.cloud/playbook/playbook:${git.triggeringCommit(scmVars)}"
```

Comment:
We're going to need to tweak 3 more files i believe in order for this to work. @gruberdev 
Please look at the following 3: 

1) https://github.com/powerhome/playbook/blob/284aea6632b3bada43394f062453376367475771/playbook-website/bin/deployer#L10

2) https://github.com/powerhome/playbook/blob/be09fa5bdc35556407d8f720acd99ed847c73f16/playbook-website/config/deploy/templates/deployment.yaml.erb#L23

3) https://github.com/powerhome/playbook/blob/be09fa5bdc35556407d8f720acd99ed847c73f16/Jenkinsfile#L22

4) https://github.com/powerhome/playbook/blob/be09fa5bdc35556407d8f720acd99ed847c73f16/Makefile#L8